### PR TITLE
Bump to 2017.1.23

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @conda-forge/core

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ build:
   always_include_files:
     - ssl/cert.pem                # [unix]
     - ssl/cacert.pem              # [unix]
-    - Library\\ssl\\cert.pem      # [win]
-    - Library\\ssl\\cacert.pem    # [win]
+    - Library/ssl/cert.pem        # [win]
+    - Library/ssl/cacert.pem      # [win]
```
Comment (scopatz) [inline recipe/meta.yaml]: Is this correct? Seems odd for windows, unless these paths are being interpreted by Python

Comment (jakirkham) [inline recipe/meta.yaml]: Yes, I think it was like that for a while, but don't remember exactly when it got that behavior.

PR ( https://github.com/conda/conda-build/pull/1280 ) that went into `conda-build` 2.0.0 changed it so as to only allow `/` even on Windows. I think this had something to do with regex support for these paths. If I left it as before, I got this [failure]( https://ci.appveyor.com/project/conda-forge/ca-certificates-feedstock/build/1.0.33/job/oqesxa7wcsunj9dq#L239 ).

Comment (scopatz) [inline recipe/meta.yaml]: makes sense.  I am happy with this PR but don't have merge right here.

Comment (jakirkham) [inline recipe/meta.yaml]: Cool.

We should give you merge rights here. Honestly all of core should have them.

Ditto with `conda-forge-build-setup` and `toolchain`. Perhaps some other things I can't think of ATM. These packages are so fundamental that we do want to make sure others can fix them if needed.

Comment (jakirkham) [inline recipe/meta.yaml]: Please see PR ( https://github.com/conda-forge/ca-certificates-feedstock/pull/13 ).

**Pull request merged**